backend/src/supabase/functions/get-room-permission/index.ts

```diff
@@ -15,27 +15,16 @@ async function returnError(msgError: string){
 }
 
 async function getPermission(supabaseClient, body) {
-  let { userId, providerId } = body;
+  let { providerId } = body;
 
   const roomData = await supabaseClient.from('rooms').select('providerId').eq('providerId', providerId);
   if (!roomData.data || (roomData.data?.length === 0))
     return returnError("No room with given data exists");
 
-  const users = await supabaseClient.from('users-data').select('userId').eq('userId', userId);
-  if (!users.data || (users.data?.length === 0))
-    return returnError("No user with given data exists");
-
-  const roomsData = await supabaseClient.from('rooms-data').select('permissionId').match({ providerId, userId });
-
-  let permission = "GUEST"
-  if (roomsData.data?.length !== 0){
-    const permissionId = parseInt(roomsData.data[0].permissionId);
-    permission = await supabaseClient.from('rooms-permission').select('name').eq( 'id', permissionId );
-    permission = permission.data[0].name;
-  }
+  const roomsData = await supabaseClient.from('rooms-data').select('id, permissionId, userEmail').eq('providerId', providerId);
```

Comment:
While we're here, should we also look up the rooms-permissions table and get the role name rather than passing on the ID to the Front-End?
This way we don't need to keep a parallel table updated in `utils/roles.js`. It will make it easier in the future if we create new roles.